### DISCUSSION TITLE
Is Valetudo ready for non-technical people?

### DISCUSSION THREAD

**Discussion by felixfischer** [Hypfer/Valetudo#1239] [Q&A / Support] [answered]
Hi there, everybody! 👋 

I bought a Dreame L10 Pro as a gift for my parents, and am thinking about replacing the original firmware with Valetudo, mainly to remove the privacy threat associated with the Xiaomi cloud.

Since I have never tried Valetudo before I wonder if this is actually a good idea. **Do you think that Valetudo is mature enough to be used by non-technical people, like my parents (~60 yo) for prolonged periods of time, without assistance?**

**I would of course do the rooting and installation myself** before giving the gift, and I will show them how to use it properly. But I fear that they could get stuck with usability issues or any kind of quirks that would make for a much poorer experience than is to be expected from the original firmware. My parents don't need any cloud functionality, just a properly working vacuum that they can operate independently at home.

**Should I do it, yay or nay?** I'd love to read your thoughts on this, or even a simple 👍 or 👎 would help me tremendously!

**Comment by Hypfer** [comment]
That is a very good question which I honestly anticipated since quite a while now

What I can say is that there have been quite a few usability improvements lately which should help here:

- The Android App to not having to deal with figuring out the IP
- The new UI being much easier to grasp
- The updater functionality not requiring you to use ssh
- Improved docs

and all the other general polishing stuff.

That being said, I can't say that the robot 100% won't for some reason nuke the /data partition, requiring you to reinstall valetudo.

I'd say probably yes?

I'd be very interested in hearing what other users think though.
I might be very biased here.

In any case, please make sure to inform your parents that this is just the unpaid hobby of some random guy on the internet.
Just for expectation management

**Comment by pegelf** [comment]
I think using valetudo is even better than using the original Xiaomi app for non-technical people. Using a local web page instead of an app has the advantage that you don't have to install anything on your smartphone, which needs to be updated frequently.
I know people can be scared of play store updates, apps which tell them that they have to accept the new terms of use or messages that tell you that your vacuum robot will stop working if you don't sacrifice your first born child within the next 5 minutes.

The robot with valetudo preinstalled as a gift will keep working like on the first day and security isn't a big issue, because everything runs in a local network. If they one day want to control the vacuum robot while they are not at home, you can maybe set them up a VPN. Also there is a timer to schedule automatic cleanups if they want to.

For controlling the vacuum on a smartphone, you have the choice to use the companion app or just create a web page shortcut which will also look like an app.
Especially for old people with bad eyes, I think it is great that you are not stuck to your smartphone. You can also use a tablet with a bigger screen or the computer with a mouse. That exists longer than phones with touch screens. So maybe your parents are more used to that.

About the features: Valetudo supports every function of the original app except the "multi floor" setting where you can save multiple maps on the same robot. But not supporting that also means a more clean UI. You also don't need that if you only have one floor. And if you have multiple floors, it is a good idea to use one robot per floor instead of carrying it around.

Since the last update, Valetudo also have some kind of "setup screen" if you don't have a wifi connection, so you could also give them the robot unconfigured and they only have to turn it on, connect to the AP of the robot, type in their wifi credentials and then everything works. I think that is easier than having to download an app, creating a Xiaomi account, receiving an SMS code, accepting to send them way too much data and so on.

Also, if your parents have a problem with using software, you can also just press the cleaning button on the robot and then it starts cleaning the whole flat without using any other device.

**Comment by jds11111** [comment]
I gave it to my spouse to test, which she doesn't normally need to use because of automation. She is very technophobic, but thought that it was pretty easy to use, and would get used to it. She runs very few apps on her phone, and was a very late adopter, so that is a pretty strong endorsement for usability. 

They will have to enter the dustbin pretty regularly for the L10.  The Z10 is self emptying, and taking out the bag would be something that they are much more accustomed to.  As long as you can check on it yourself every couple of months, I think that they would be fine.

**Comment by felixfischer** [comment, marked as answer]
I can now report that the answer to my question is YES! My non-techy parents are happily using their Valetudo powered vacuum in their flat, without my assistance :)  The only (very minor) issue is that the interface is only in English. But they already know what buttons to push, so they're good!

**Thank you for making Valetudo! It's absolutely awesome! 🥇**
